### DISCUSSION TITLE
Error while loading query results: unable to open database file

### DISCUSSION THREAD

**Discussion by lkhatkar** [pawelsalawa/sqlitestudio#4238] [General]
### Details
Result windows throws error on clicking on next page button from 3rd page onwards.

### Steps to reproduce
Run select query. (result count should be >2000 records)
Query result window shows >2 pages with 1000 records in each page.
Click on next page button (it works for first 2 pages and then throws error)

### Operating system
Windows 10, AWS machine.

### SQLiteStudio version
3.2.1

**Comment by pawelsalawa** [comment]
What is the file system? Is is attached or network drive?
Be aware that SQLite official site says it's not good idea and you may expect such problems: https://www.sqlite.org/draft/useovernet.html

**Comment by lkhatkar** [comment]
it is attached

**Reply by pawelsalawa** [reply]
Then the limitations mentioned in the link above apply to your case.

**Reply by lkhatkar** [reply]
Thats ok, Is there any solution?
issue is only with gui, it is affecting export functionality.
If you can suggest some alternative way to export all records in csv file it will be very helpful to me.

Thank you so much

**Reply by pawelsalawa** [reply]
Are you able to export successfully the same amount of data & objects using command line?

**Reply by lkhatkar** [reply]
Ok, I will give it a try.

**Reply by lkhatkar** [reply]
It worked!

followed example: $ sqlite3 -header -csv your_database_name "select * from books;" > books.csv
Thank you
